rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,139 @@
+The stored manifest file could then be used in the HCA CLI to download the exact versions of the data and metadata for that version of the project. It is a requirement that the DCP never deletes older versions of cited data files, except for the special case of retraction of unconsented data.
+
+#### Possible DOI Registration Agencies and open-access repositories that provide the ability to assign DOIs:
+  - [BioStudies](https://www.ebi.ac.uk/biostudies/)
+  - [Zenodo](https://en.wikipedia.org/wiki/Zenodo)
+  - [Figshare](https://en.wikipedia.org/wiki/Figshare)
+  - [Crossref](https://en.wikipedia.org/wiki/Crossref)
+  - HCA DCP assigns its own DOIs by becoming a member of the International DOI Foundation (IDF)
+
+#### DOI Versioning
+Most DOI repositories appear to provide support for versioning (i.e. multiple versions of a single citation available e.g. via a drop-down selection),
+the ability to store accompanying files in the DOI repository (this would be useful for storing versions of manifest and metadata tsv files, etc.),
+and the ability to store URL references (useful for linking back to a page in the Data Browser).
+Examining how both Figshare and Zenodo do DOIs, as the group minting DOIs controls everything after the prefix it is possible to have a main DOI which points to the most recent version and then versioned URLs which point to specific versions.
+
+#### Data Browser
+The Data Browser would contain a reference to the base DOI for each project. Clicking on the DOI link would redirect the user to the (external) DOI repository where the user could see all the versions of that project and download the associated manifest and metadata.tsv files for a specific version. 
+
+#### DOI Creation/Update Process
+Embedding a DOI in the metadata (e.g. biostudies_doi) during ingest would provide per-project citability.
+Provided the DOI repository supports versioning the metadata DOI field could be updated automatically by Ingest to a new version whenever a project update is processed.
+Using a DOI may be the simplest way to provide versioned views of a project.
+
+The creation/update process would perform the following steps:
+  - Ingest creates a new DOI (new project) or a new version of an existing DOI (updated project) when the submission is deemed “complete”.
+  - Upload the corresponding metadata tsv and manifest file for this version of the project to the DOI repository (need to wait for these to be generated).
+  - Upload the matrix output file(s) for this version of the project to the DOI repository.
+  - Update a project description to the DOI repository (e.g. what’s in this version?).
+  - Create a link in the DOI repository back to the project details page in the Data Browser.
+
+### Acceptance Criteria [optional]
+
```

Comment:
Please add acceptance criteria for the three phases and success metrics that we can use to see if our solutions actually work for the community

Reply:
Added acceptance criteria for each phase - please review.

Reply:
@lauraclarke Are the acceptance criteria, err, acceptable?

Reply:
Thanks for the acceptance criteria, these look good but the phase 3 criteria seem disconnected from the user stories as they are written now. The user stories don't mention a data release

Reply:
Added Data Release User Story.